Index: libxext/man/DBE.man
===================================================================
--- libxext.orig/man/DBE.man	2006-08-26 18:13:16.000000000 +0000
+++ libxext/man/DBE.man	2006-08-26 18:13:22.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/DBE.man,v 3.4 2001/01/27 18:20:19 dawes Exp $
 .\"
-.TH DBE 3X11 __xorgversion__ "X FUNCTIONS"
+.TH DBE __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 DBE - Double Buffer Extension
 .SH SYNOPSIS
Index: libxext/man/XShape.man
===================================================================
--- libxext.orig/man/XShape.man	2006-08-26 18:13:04.000000000 +0000
+++ libxext/man/XShape.man	2006-08-26 18:13:10.000000000 +0000
@@ -32,7 +32,7 @@
 .ie t \fB\^\\$1\^\fR\\$2
 .el \fI\^\\$1\^\fP\\$2
 ..
-.TH XSHAPE 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XSHAPE __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XShapeQueryExtension, XShapeQueryVersion, XShapeCombineRegion, XShapeCombineRectangles, XShapeCombineMask, XShapeCombineShape, XShapeOffsetShape, XShapeQueryExtents, XShapeSelectInput, XShapeInputSelected, XShapeGetRectangles \- X nonrectangular shape functions
 .SH SYNTAX
Index: libxext/man/XcupGetReservedColormapEntries.man
===================================================================
--- libxext.orig/man/XcupGetReservedColormapEntries.man	2006-08-26 18:07:48.000000000 +0000
+++ libxext/man/XcupGetReservedColormapEntries.man	2006-08-26 18:08:28.000000000 +0000
@@ -33,7 +33,7 @@
 .ie t \fB\^\\$1\^\fR\\$2
 .el \fI\^\\$1\^\fP\\$2
 ..
-.TH XCUP 3Xext __xorgversion__ "X FUNCTIONS"
+.TH XCUP __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XcupGetReservedColormapEntries \- list colormap entries reserved by the system
 .SH SYNOPSIS
Index: libxext/man/XcupQueryVersion.man
===================================================================
--- libxext.orig/man/XcupQueryVersion.man	2006-08-26 18:08:43.000000000 +0000
+++ libxext/man/XcupQueryVersion.man	2006-08-26 18:08:54.000000000 +0000
@@ -33,7 +33,7 @@
 .ie t \fB\^\\$1\^\fR\\$2
 .el \fI\^\\$1\^\fP\\$2
 ..
-.TH XcupQueryVersion 3Xext __xorgversion__ "X FUNCTIONS"
+.TH XcupQueryVersion __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XcupQueryVersion \- Returns TOG-CUP protocol version supported by the X server
 .SH SYNOPSIS
Index: libxext/man/XcupStoreColors.man
===================================================================
--- libxext.orig/man/XcupStoreColors.man	2006-08-26 18:09:02.000000000 +0000
+++ libxext/man/XcupStoreColors.man	2006-08-26 18:09:28.000000000 +0000
@@ -33,7 +33,7 @@
 .ie t \fB\^\\$1\^\fR\\$2
 .el \fI\^\\$1\^\fP\\$2
 ..
-.TH XcupStoreColors 3Xext __xorgversion__ "X FUNCTIONS"
+.TH XcupStoreColors __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XcupStoreColors \- initialize shareable colormap entries at specific locations
 .SH SYNOPSIS
Index: libxext/man/XdbeAllocateBackBufferName.man
===================================================================
--- libxext.orig/man/XdbeAllocateBackBufferName.man	2006-08-26 18:10:56.000000000 +0000
+++ libxext/man/XdbeAllocateBackBufferName.man	2006-08-26 18:11:05.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeAllo.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeAllocateBackBufferName 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeAllocateBackBufferName __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeAllocateBackBufferName - allocates a DBE buffer.
 .SH SYNOPSIS
Index: libxext/man/XdbeBeginIdiom.man
===================================================================
--- libxext.orig/man/XdbeBeginIdiom.man	2006-08-26 18:11:11.000000000 +0000
+++ libxext/man/XdbeBeginIdiom.man	2006-08-26 18:11:17.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeBegi.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH DBE 3X11 __xorgversion__ "X FUNCTIONS"
+.TH DBE __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeBeginIdiom - marks the beginning of a DBE idiom sequence.
 .SH SYNOPSIS
Index: libxext/man/XdbeDeallocateBackBufferName.man
===================================================================
--- libxext.orig/man/XdbeDeallocateBackBufferName.man	2006-08-26 18:11:24.000000000 +0000
+++ libxext/man/XdbeDeallocateBackBufferName.man	2006-08-26 18:11:31.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeDeal.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeDeallocateBackBufferName 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeDeallocateBackBufferName __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeDeallocateBackBufferName - frees a DBE buffer.
 .SH SYNOPSIS
Index: libxext/man/XdbeEndIdiom.man
===================================================================
--- libxext.orig/man/XdbeEndIdiom.man	2006-08-26 18:11:38.000000000 +0000
+++ libxext/man/XdbeEndIdiom.man	2006-08-26 18:11:46.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeEndI.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeEndIdiom 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeEndIdiom __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeEndIdiom - marks the end of a DBE idiom sequence.
 .SH SYNOPSIS
Index: libxext/man/XdbeFreeVisualInfo.man
===================================================================
--- libxext.orig/man/XdbeFreeVisualInfo.man	2006-08-26 18:11:51.000000000 +0000
+++ libxext/man/XdbeFreeVisualInfo.man	2006-08-26 18:11:57.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeFree.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeFreeVisualInfo 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeFreeVisualInfo __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeFreeVisualInfo - frees information returned by
 .B XdbeGetVisualInfo().
Index: libxext/man/XdbeGetBackBufferAttributes.man
===================================================================
--- libxext.orig/man/XdbeGetBackBufferAttributes.man	2006-08-26 18:13:33.000000000 +0000
+++ libxext/man/XdbeGetBackBufferAttributes.man	2006-08-26 18:13:38.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeGetB.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeGetBackBufferAttributes 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeGetBackBufferAttributes __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeGetBackBufferAttributes - returns attributes of a DBE buffer.
 .SH SYNOPSIS
Index: libxext/man/XdbeGetVisualInfo.man
===================================================================
--- libxext.orig/man/XdbeGetVisualInfo.man	2006-08-26 18:12:04.000000000 +0000
+++ libxext/man/XdbeGetVisualInfo.man	2006-08-26 18:12:10.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeGetV.man,v 3.5 2001/02/07 22:35:22 tsi Exp $
 .\"
-.TH XdbeGetVisualInfo 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeGetVisualInfo __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeGetVisualInfo - Get dbe Visual Informations
 .SH SYNOPSIS
Index: libxext/man/XdbeQueryExtension.man
===================================================================
--- libxext.orig/man/XdbeQueryExtension.man	2006-08-26 18:12:16.000000000 +0000
+++ libxext/man/XdbeQueryExtension.man	2006-08-26 18:12:21.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeQuer.man,v 3.4 2001/01/27 18:20:20 dawes Exp $
 .\"
-.TH XdbeQueryExtension 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeQueryExtension __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeQueryExtension - returns the version of DBE supported by the server.
 .SH SYNOPSIS
Index: libxext/man/XdbeSwapBuffers.man
===================================================================
--- libxext.orig/man/XdbeSwapBuffers.man	2006-08-26 18:12:30.000000000 +0000
+++ libxext/man/XdbeSwapBuffers.man	2006-08-26 18:12:35.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/dbe/XdbeSwap.man,v 3.5 2001/02/07 22:35:22 tsi Exp $
 .\"
-.TH XdbeSwapBuffers 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XdbeSwapBuffers __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XdbeSwapBuffers - swaps front and back DBE buffers.
 .SH SYNOPSIS
Index: libxext/man/Xevi.man
===================================================================
--- libxext.orig/man/Xevi.man	2006-08-26 18:12:40.000000000 +0000
+++ libxext/man/Xevi.man	2006-08-26 18:12:45.000000000 +0000
@@ -27,7 +27,7 @@
 .\"
 .\" $XFree86: xc/doc/man/Xext/Xevi.man,v 1.2 2001/01/27 18:20:19 dawes Exp $
 .\"
-.TH XEVI 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XEVI __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XeviQueryExtension, XeviQueryVersion, XeviGetVisualInfo - X Extended Visual Information functions
 .SH SYNOPSIS
Index: libxext/man/Xmbuf.man
===================================================================
--- libxext.orig/man/Xmbuf.man	2006-08-26 18:12:53.000000000 +0000
+++ libxext/man/Xmbuf.man	2006-08-26 18:12:59.000000000 +0000
@@ -41,7 +41,7 @@
 .ie t \fB\^\\$1\^\fR\\$2
 .el \fI\^\\$1\^\fP\\$2
 ..
-.TH XMBUF 3X11 __xorgversion__ "X FUNCTIONS"
+.TH XMBUF __libmansuffix__ __xorgversion__ "X FUNCTIONS"
 .SH NAME
 XmbufQueryExtension, XmbufGetVersion, XmbufCreateBuffers, XmbufDestroyBuffers, XmbufDisplayBuffers, XmbufGetWindowAttributes, XmbufChangeWindowAttributes, XmbufGetBufferAttributes, XmbufChangeBufferAttributes, XmbufGetScreenInfo, XmbufCreateStereoWindow - X multibuffering functions
 .SH SYNTAX
